src/sentry/static/sentry/app/utils/withIssueTags.tsx

```diff
@@ -72,17 +89,20 @@ const withIssueTags = <P extends InjectedTagsProps>(
     },
 
     setAssigned() {
-      if (this.state.users && this.state.tags.assigned) {
-        const {tags, users} = this.state;
+      if (this.state.tags.assigned) {
```

Comment:
do need to check for `this.state.users` and `this.state.teams` now?

Reply:
Ah, you're right - removed the check 👍